devops/deploy/spin-up-script.sh

```diff
@@ -1,25 +1,46 @@
 #!/bin/bash
 
+# Just a basic script to spin up listening servers on different ports in dynamic port range
+# Setting up variables
 PSTART=${1:-50000}
 PEND=${2:-50050}
-# Just a basic script to spin up listening servers on different ports in dynamic port range
+PEER_FILE="peer_list.txt"
 IP=127.0.0.1
 COUNT=0
 ((NUM_PORTS=${PEND}-${PSTART}))
 
+# Making keys directory to hold public/private keys if not exist
+if [ -d keys/ ]
+then
+	echo "Directory keys exists."
+else
+	mkdir keys/
+fi
+
```

Comment:
Would be good to create the `peer_list.txt` file here and in, at least a minor attempt at some security, `chmod` it to `600`.